docs/topics/introduction/quickstart.rst

```diff
@@ -3,12 +3,12 @@
 Quick Start
 ===========
 
-*This guide does not go into detail as to how everything works, but hopefully will get you scraping quickly. For more information about each process works please see the rest of the documentation.*
+*This guide does not go into detail as to how everything works but, hopefully, it will get you scraping quickly. For more information about how each process works please see the rest of the documentation.*
```

Comment:
I agree with all changes except this line, if you revert this back to 
```
....how everything works, but hopefully will get you scraping....
```
I can accept. Adding a pause after `but` doesn't flow right in my mind. If we need to change the sentence up to remove that, perhaps that's the better option.

Reply:
@madisonb what about 

``` ... to how everything works but it will hopefully get you scraping quickly. ``` ?

Reply:
that works

Reply:
[Done](https://github.com/istresearch/scrapy-cluster/pull/256/commits/05ce90c0f63cde8a4b19ee332ff258703393a468).